sql/core/src/test/scala/org/apache/spark/sql/CsvFunctionsSuite.scala

```diff
@@ -574,4 +574,36 @@ class CsvFunctionsSuite extends QueryTest with SharedSparkSession {
       $"csv", schema_of_csv("1,2\n2"), Map.empty[String, String].asJava))
     checkAnswer(actual, Row(Row(1, "2\n2")))
   }
+
+  test("SPARK-41049: make to_csv function deterministic") {
```

Comment:
I think this test is already covered by https://github.com/apache/spark/pull/39248/files#diff-0b6750da4fdbe797612d14fa705138892221bca66e748241ff390b58ab4e5352R3571

Reply:
removed